RecoTauTag/RecoTau/python/RecoTauCombinatoricProducer_cfi.py

```diff
@@ -137,7 +137,7 @@
     jetRegionSrc = cms.InputTag("recoTauAK4PFJets08Region"),
     chargedHadronSrc = cms.InputTag('ak4PFJetsRecoTauChargedHadrons'),                                
     piZeroSrc = cms.InputTag("ak4PFJetsRecoTauPiZeros"),
-    buildNullTaus = cms.bool(True),
+    buildNullTaus = cms.bool(False),
```

Comment:
is this supposed to resolve this issue https://github.com/slava77/cmssw/issues/99 ?